packs/cisco_umbrella_dns.yml

```diff
@@ -0,0 +1,10 @@
+AnalysisType: pack
+PackID: PantherManaged.Cisco.Umbrella
+Description: Group of all Cisco Umbrella DNS detections
```

Comment:
Wondering if this should be the following (for any detections we add in the future that aren't `DNS` specifically)

```suggestion
Description: Group of all Cisco Umbrella detections
```

Reply:
Makes sense to me